src/shared/components/oncoprint/SortUtils.ts

```diff
@@ -0,0 +1,176 @@
+import {TrackSortComparator} from "oncoprintjs";
+import {ClinicalTrackSpec, GeneticTrackDatum} from "./Oncoprint";
+function makeComparatorMetric(array_spec:(string|string[]|undefined|boolean)[]) {
+    let metric:{[s:string]:number} = {};
+    for (let i=0; i<array_spec.length; i++) {
+        const equiv_values = ([] as any[]).concat(array_spec[i]);
+        for (let j=0; j<equiv_values.length; j++) {
+            metric[equiv_values[j]] = i;
+        }
+    }
+    return metric;
+};
+
+function sign(x:number):0|-1|1 {
```

Comment:
duplicate?

Reply:
strangely enough a grep for "function sign" and "function sgn" dont turn up anything